docs/reference/technology/mesh/quick_start_hw.md

```diff
@@ -45,6 +45,16 @@ The following table shows which development boards have been tested. It does not
 | Onsemi NCS36510 <span style='background-color: #5f5;'>(internal RF)</span> | | | |
 | NXP KW24D <span style='background-color: #5f5;'>(internal RF)</span> | | <span style='background-color: #f00;'>Yes **(4)**</span> | |
 
+#### File System
+
+Thread network stack can save network configurations settings to file system once:
+
+- You enable the file system as instructed in the [Mbed OS storage documentation](https://os.mbed.com/docs/latest/reference/storage.html).
+- You set the file system root path to the Thread network stack by calling the function `ns_file_system_set_root_path(root-path)`. Set the file system root path before starting the Thread stack to read possible configuration settings in the first power up.
```

Comment:
"Set the file system root path" can be replaced with "do this"; it's shorter and makes it clearer this is all one action